omniport/config/webpack.dev.js

```diff
@@ -28,6 +28,7 @@ module.exports = merge(common, {
         '/session_auth',
         '/open_auth',
         '/token_auth',
+	'/guest_auth',
```

Comment:
@rish9101 The indentation is a little off.

Reply:
A little? It's not going work with that indentation afaik.

Reply:
@dhruvkb it's JS. Everything will work 😂

Reply:
My bad. I confused that with Python 😂!